### PR TITLE
Added Go Export Others templates.

### PR DESCRIPTION
**Pull request by ljades**

### CODE REVIEW

File: fixtures/data_criteria/care_goal.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+    {
+        "title": "Repurposed Symptom Active for Testing",
```
Comment (mattrbianchi) [inline fixtures/data_criteria/care_goal.json]: This json file and the ones below it all have some inconsistent indentation combined with uneven braces. I would prefer pure TABS to avoid this issue, but at least fix the files and make them consistent. It's hard to follow what fields are in what objects with inconsistent style.

**Review by mattrbianchi — CHANGES_REQUESTED** [review]

**Review by mattrbianchi — APPROVED** [review]

File: exporter/templates_test.go [hunk 2]
```diff
@@ -835,14 +905,14 @@ func printXmlString(xmlString string) {
...
-	printXmlString(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
+	//printXmlString(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
 	return xmlRootNode(generateXML(fileName, getDataForQrdaOid(qrdaOid)))
 }
 
 // same as xmlRootNodeForQrdaOid() function but allows custom input data (should be an EntryInfo struct)
 func xmlRootNodeForQrdaOidWithData(qrdaOid string, data interface{}) *xml.ElementNode {
 	fileName := "_" + qrdaOid + ".xml"
-	printXmlString(generateXML(fileName, data))
+	//printXmlString(generateXML(fileName, data))
```
Comment (ghost) [inline exporter/templates_test.go:915]: Is there a reason this is commented instead of removed? otherwise I'm good to merge this in.

Comment (ljades) [reply exporter/templates_test.go:915]: Generally we leave them there so it's a readily available (and readably easy) option to uncomment them for debugging. But I think we generally try to have the master branch have the commented versions, which is why I did that.
